order/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db      import models
+from user.models    import Users
+from product.models import Products
+
+class Payments(models.Model):
+    user       = models.models.ForeignKey(Users, on_delete=models.CASCADE)
+    card       = models.CharField(max_length=50)
+    createtime = models.DateField(auto_now_add=True)
+
+class Orders(models.Model):
+    status     = models.ForeignKey("Status", on_delete=models.CASCADE)
+    payment    = models.ForeignKey("Payments", on_delete=models.CASCADE)
+    address    = models.ForeignKey("Address", on_delete=models.CASCADE)
+
+class Status(models.Model):
+    name = models.CharField(max_length=50)
+
+class Carts(models.Model):
+    user         = models.ForeignKey(Users, on_delete=models.CASCADE)
+    product      = models.ForeignKey(Products, on_delete=models.CASCADE)
+    order        = models.ForeignKey("Payments", on_delete=models.CASCADE)
+    count        = models.IntegerField(default=1)
+    giftwrapping = models.BooleanField()
```

Comment:
user는 order를 참조하고 있기 때문에 여기서도 참조할 필요가 없습니다.
```suggestion
class Carts(models.Model):
    order        = models.ForeignKey("Payments", on_delete=models.CASCADE)
    product      = models.ForeignKey(Products, on_delete=models.CASCADE)
    count        = models.IntegerField(default=1)
    giftwrapping = models.BooleanField()
```